### PR TITLE
Updates Hub components name by adding prefix as `tekton-hub`

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

**Comment by PuneetPunamiya** [comment]
/hold
Keeping the patch in hold till we have Hub 1.7 released

File: pkg/apis/operator/v1alpha1/tektonhub_defaults.go [hunk 1]
```diff
@@ -24,11 +24,11 @@ import (
 func (th *TektonHub) SetDefaults(ctx context.Context) {
 
 	if th.Spec.Db.DbSecretName == "" {
-		th.Spec.Db.DbSecretName = "db"
+		th.Spec.Db.DbSecretName = "tekton-hub-db"
```
Comment (nikhil-thomas) [inline pkg/apis/operator/v1alpha1/tektonhub_defaults.go:27]: let us move the string `tekton-hub-` into a project level constant or a package level constant.

**Comment by PuneetPunamiya** [comment]
retest

**Comment by nikhil-thomas** [comment]
@PuneetPunamiya  could you look into this 
```
fetching | component: hub | version: v1.7.0
https://github.com/tektoncd/hub/releases/download/v1.7.0/db.yaml
url: https://github.com/tektoncd/hub/releases/download/v1.7.0/db.yaml
./hack/fetch-releases.sh: line 154: comp: unbound variable
```
https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_operator/654/pull-tekton-operator-integration-tests/1500812614161141760/

**Comment by PuneetPunamiya** [comment]
> @PuneetPunamiya could you look into this
> 
> ```
> fetching | component: hub | version: v1.7.0
> https://github.com/tektoncd/hub/releases/download/v1.7.0/db.yaml
> url: https://github.com/tektoncd/hub/releases/download/v1.7.0/db.yaml
> ./hack/fetch-releases.sh: line 154: comp: unbound variable
> ```
> 
> https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_operator/654/pull-tekton-operator-integration-tests/1500812614161141760/

Yeah i have updated the pr, actually i forgot that hub 1.7.0 was not release, now since it is released this should pass

**Pull request merged**